datafusion/expr/src/expr.rs

```diff
@@ -324,6 +319,20 @@ impl Like {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, Hash)]
```

Comment:
Overlooked it because I wasn't sure what expression `GetIndexedField` corresponds to, but should add a comment here explaining the purpose of this struct